handler/callhandler.cpp

```diff
@@ -353,6 +355,7 @@ void CallHandler::onCallChannelInvalidated()
     ToneGenerator::instance()->stopTone();
     if (mCallChannels.isEmpty() && !mHangupRequested) {
         ToneGenerator::instance()->playCallEndedTone();
+		std::this_thread::sleep_for(std::chrono::milliseconds(2200));
```

Comment:
Not sure if blocking the whole event loop is the best thing to do in this case, seems like canonical added a timer here https://github.com/ubports/telephony-service/blob/xenial/handler/audioroutemanager.cpp#L46 to cover exactly this problem, maybe its better to increase that (for now)?

But i would much more prefer a std::atomic<bool> to be tell the audioroutemanager when it can and cannot change the audio output. Maybe in ToneGenerator would be the best place for this.

Reply:
True lets try to increase that timer, it seems just to be a bit too short...